.github/workflows/dockerimage.yml

```diff
@@ -2,22 +2,40 @@ name: Docker Image CI
 
 on:
   release:
-    branches: [ master ]    
+    branches: [ master ]
 
 jobs:
-
   build:
-
     runs-on: ubuntu-latest
-
     steps:
-      - uses: actions/checkout@ee0669bd1cc54295c223e0bb666b733df41de1c5 # v2.7.0
-      - uses: docker/build-push-action@3e7a4f6646880c6f63758d73ac32392d323eaf8f # v1.1.2
+      -
+        name: Checkout
+        uses: actions/checkout@v3
```

Comment:
could you also pin the actions to hashes instead of versions like they used to be, as this further mitigates the risks regarding the release workflow.

Reply:
It's a good point. I forgot to pin the versions to patch versions. Very happy to do that. 

One follow up question though. I believe it is possible to use the semantic versions directly, e.g., `uses: actions/checkout@v3.2.1`,  instead of using hashes. Do you have a preference on which to use?

Reply:
hashes are preferred to semantic versions, as hashes are static. You can use https://app.stepsecurity.io/secureworkflow/ to easily pin the hashes based on the versions supplied, instead of having to dig for them manually :)

Reply:
Cool. Makes sense. 

That's done now.